biz.aQute.bndlib.tests/src/test/ResourceTest.java

```diff
@@ -204,7 +206,7 @@ public void testResourceToVersionedClause() throws Exception {
 		VersionedClause versionClause = ResourceUtils.toVersionClause(resource, "[===,==+)");
 		StringBuilder sb = new StringBuilder();
 		versionClause.formatTo(sb);
-		assertEquals("demo-fragment;version='[3.3.0,3.3.1)'", sb.toString());
+		assertEquals("demo-fragment;version='[3.4.0,3.4.1)'", sb.toString());
```

Comment:
I am thinking the demo-fragment project should set it's version to a fixed 1.0 version so that each time we need to rebuild demo-fragment, we don't have to change this assert.